### DISCUSSION TITLE
Buff Chevalier

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#2137] [Nouvelles idées]
Bonjour tout le monde ! 

Depuis la dernière mise à jour, j'ai collecté des stats sur les combats dans l'arène =D
Les données sont composés de 98 combats réalisés uniquement par les joueurs du top15 de la gloire. J'ai écarté les égalités (vétéran principalement ;) ), ainsi que les match amicaux.. 

Ensuite, j'ai réalisé un modèle statistique prenant en compte la différence de niveau entre les joueurs et la classe jouée. 
Voici mes résultats :

En tenant compte de la différence de niveau, voici les proba de win par classe :
Chevalier : 11.78 % *
Mage : 51.42%
Paladin : 41.29%
Cano : 51.45%
Vétéran : 76.91% *

La différence de niveau est un facteur très significatif et seules deux classes sont statistiquement différentes du cano : Vétéran et Chevalier, avec tous les deux un p-valeur à 6% !! Et il y a une chance sur trois que le paladin soit équivalent au cano..
 J'ai écarté des données du Fantassin et du Tank trop peu joué en arène.. (J'ai quand même joué le fantassin 1 mois avant la maj et je n'ai gagné qu'un match amical avec +20 level d'écart..)
 
 **Statistiquement, il faudrait 33 niveaux d'écart pour qu'un Chevalier soit équivalent à un Canonnier, elle n'est plus viable.** 
 
 Ma proposition est la suivante : 
réduire les points de vie du Chevalier de 90 et restaurer ses anciennes capacités.
 
 Cette classe était déséquilibré au début de la mise à jour sur les arènes car les autres joueurs étaient peu adaptés. Je trouve aussi que c'était très sympa cette rivalité cano avec égide puissante contre Chevalier, il contribuait positivement au mécanisme pierre-feuille-ciseau..

**Comment by romain22222** [comment]
Coucou easy, je voulais juste souligner aussi que le % de winrate du vétéran est si haut car je suis l'un des seuls à jouer cette classe, en plus de la jouer quasi parfaitement. Le chevalier par contre, est joué par plus de gens et a subit un gros nerf. Il faut je pense attendre que ca se tasse voir comment va évoluer les %ages. Enfin, en prenant que le top niveau (les 15 1ers en gloire) tu ignores le reste complet du tableau en terme de %age, ne serait ce que ceux qui jouent bien plus casuellement. Je pense qu'un vétéran qui ne sait pas jouer va se faire exploser par un chevalier (ne serait ce qu'il utilise la concentration :doge:). Le buff proposé est de toute facon trop fort, le problème du chevalier n'est pas spécialement sa vie, c'était qu'il avait trop d'options pour gagner / faire nulle le match. Pour la rendre équilibré, il faut SURTOUT pas revenir en arrière, la classe avant était juste trop puissante.

**Reply by BastLast** [reply]
Un buff du chevalier est prévu, il serait intéressant d'avoir des trucs sur grafana @romain22222 si tu as le temps ça m'aidera bcp dans l'équilibrage

**Comment by Easygamer01s** [comment]
Sur la période, il y a eu trois vétérans différent.. Je pense être le dernier chevalier xD En dehors de quelques petites erreurs parfois, je pense que le top15 sait jouer sa classe =D
J'ajouterais aussi que les stats de win ne font pas tout, il y a aussi la profondeur de la classe qui être prise en compte.. Le chevalier se résume beaucoup à spammer l'attaque simple.. Vu que le marteau ne peut plus être utilisé de la même façon qu'avant, il est très rare, voir impossible, d'aller chercher l'égalité en fin de match..

Je proposais une réduction de sa vie car je pense que c'est comme cela que vous souhaitez compenser la force d'une classe par rapport à celle d'une autre.. D'ailleurs le mage est un excellent choix vu que +- 50% de win avec beaucoup plus de pv que le canno =D

**Reply by BastLast** [reply]
Btw on a deja toutes les stats que tu t'es embêté a faire 😅
